### ISSUE TITLE
Execution failed for task ':app:clearScreenshots'. > A problem occurred starting process 'command 'adb''

### ISSUE THREAD

**Issue by fangzhzh** [facebook/screenshot-tests-for-android#5]
I use  0.2.3
```
classpath 'com.facebook.testing.screenshot:plugin:0.2.3'
```

When I run
```
$./gradlew --stacktrace screenshotTests
```
It gave me wrong message like 'can't find adb', but I'm pretty sure my adb is in PATH.
```
Parallel execution is an incubating feature.
{filter=org.gradle.api.artifacts.maven.PublishFilter$1@11166e59, snapshotRepository=null, pomFilterContainer=org.gradle.api.publication.maven.internal.BasePomFilterContainer@78f561fe, repository=null, name=null, activePomFilters=[org.gradle.api.publication.maven.internal.DefaultPomFilter@60227fd0], settings=org.gradle.mvn3.org.apache.maven.settings.Settings@35f4a8bc, class=class org.gradle.api.publication.maven.internal.deployer.DefaultGroovyMavenDeployer, configuration=null, uniqueVersion=true, artifactPomContainer=org.gradle.api.publication.maven.internal.DefaultArtifactPomContainer@5171ea50, pom=org.gradle.api.publication.maven.internal.pom.DefaultMavenPom@dce53f6}
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
WARNING [Project: :app] To shrink resources you must also enable ProGuard
:app:clearScreenshots
:protocol:compileLint
:appkit:compileLint
:protocol:copyReleaseLint UP-TO-DATE
:protocol:preBuild
:appkit:copyReleaseLint UP-TO-DATE
:protocol:preBuild UP-TO-DATE
:appkit:preBuild
:protocol:preReleaseBuild UP-TO-DATE
:appkit:preBuild UP-TO-DATE
:appkit:preReleaseBuild UP-TO-DATE
:app:clearScreenshots FAILED
:appkit:checkReleaseManifest

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:clearScreenshots'.
> A problem occurred starting process 'command 'adb''

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:clearScreenshots'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:69)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:46)
	at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExecuter.execute(PostExecutionAnalysisTaskExecuter.java:35)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:64)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:42)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:53)
	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
	at org.gradle.api.internal.AbstractTask.executeWithoutThrowingTaskFailure(AbstractTask.java:310)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.executeTask(AbstractTaskPlanExecutor.java:79)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:63)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:51)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
	at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: org.gradle.process.internal.ExecException: A problem occurred starting process 'command 'adb''
	at org.gradle.process.internal.DefaultExecHandle.setEndStateInfo(DefaultExecHandle.java:196)
	at org.gradle.process.internal.DefaultExecHandle.failed(DefaultExecHandle.java:325)
	at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:83)
	... 2 more
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'adb'
	at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
	at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
	at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:65)
	... 2 more
Caused by: java.io.IOException: Cannot run program "adb" (in directory "/Users/zhenfangzhang/android/example/app"): error=2, No such file or directory
	at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
	... 4 more
Caused by: java.io.IOException: error=2, No such file or directory
	... 5 more


BUILD FAILED

Total time: 2.037 secs
```

**Comment by fangzhzh** [comment]
Environment: OS X 10.11, Android studio

I found "cearScreenshopts" use project.screenshots.adb, so I configured it at my gradle file.

```
screenshots {
    testApkTarget = 'packageFlavar1DebugAndroidTest'
    adb = '/Users/zhenfangzhang/Library/Android/sdk/platform-tools/adb'
}
```
In case someone would encounter same problem, I will leave this issue  here and will close it.
In case some one

**Comment by tdrhq** [comment]
ooh this is a bad experience, I should automatically detect adb from ANDROID_SDK (already doing that in some of the python ncode)

**Comment by tdrhq** [comment]
This got fixed in 74ffec6e05f1d44f7d176b7232e59c536fcc84bb , I'll bump version next week since this is not urgent.
